handbook/company.md

```diff
@@ -101,7 +101,8 @@ Empathy leads to better understanding, better communication, and better decision
   > At Fleet, we think about the customer first.  No matter what kind of work you are doing, you serve the customer.
   > When customers buy Fleet, they trust us to provide the solution for important problems.
   > Imagine you are the person making the decision to buy Fleet.  Imagine you are responsible for all of your organization's laptops and servers.  Imagine you log in to the product every day.  Imagine you are the developer writing code to integrate with Fleet's REST API.  Imagine you are deploying the server, or running the upgrade scripts.  Imagine you are responsible for keeping your organization's computers secure and running smoothly.
-  > You would rest easier, knowing that everyone who works at Fleet is seeking to deliver the experience they would want for themselves, in your shoes. --> 
+  > You would rest easier, knowing that everyone who works at Fleet is seeking to deliver the experience they would want for themselves, in your shoes. 
```

Comment:
FYI @Desmi-Dizney this would not have worked -- you have to get rid of everything between `<!--` and `-->` (see the Loom video I linked in Slack for more info.)